.github/workflows/databend-release.yml

```diff
@@ -346,6 +346,52 @@ jobs:
           file: ./docker/${{ matrix.distro }}/${{ matrix.service }}.Dockerfile
           build-args: VERSION=${{ needs.create_release.outputs.version }}
 
+  release_packages:
+    name: release packages
+    runs-on: ubuntu-latest
+    needs: [create_release, publish_linux]
+    strategy:
+      matrix:
+        arch:
+          - x86_64
+        platform:
+          - musl
```

Comment:
maybe we should release gnu build?

Reply:
emmm, let's start by confirming that this workflow is correct and that switching targets is easy.